test/kuzzle/listenersManagement.test.js

```diff
@@ -28,46 +28,50 @@ describe('Listeners management', () => {
 
   describe('#addListener', function () {
     it('should properly add new listeners to events', function () {
-      should(kuzzle.eventListeners['connected'].listeners.length).be.exactly(4);
-      should(kuzzle.eventListeners['error'].listeners.length).be.exactly(3);
-      should(kuzzle.eventListeners['disconnected'].listeners.length).be.exactly(2);
-      should(kuzzle.eventListeners['reconnected'].listeners.length).be.exactly(1);
+      should(kuzzle.eventListeners.connected.listeners.length).be.exactly(4);
+      should(kuzzle.eventListeners.error.listeners.length).be.exactly(3);
+      should(kuzzle.eventListeners.disconnected.listeners.length).be.exactly(2);
+      should(kuzzle.eventListeners.reconnected.listeners.length).be.exactly(1);
     });
 
     it('should throw an error if trying to adding a listener to an unknown event', function () {
       try {
         kuzzle.addListener('foo', function () {});
         should.fail('success', 'failure', 'Should have failed to add a listener to unknown event "foo"', '');
       }
-      catch (e) { }
+      catch (e) {
+        // done
+      }
     });
 
     it('should throw an error when providing a non-function listener argument', function () {
       try {
         kuzzle.addListener('connected', 'bar');
         should.fail('success', 'failure', 'Should have failed to add a string listener', '');
```

Comment:
I think I wrote that code. I had discovered `should.fail` existence and I wanted to play with it, instead of adding a callback to `it()` and invoke it with an error if the expected throw did not happen.